### DISCUSSION TITLE
Bug: 404 error - checkout link invalid or expired

### DISCUSSION THREAD

**Discussion by vettloffah** [swellstores/community#94] [Help]
We are experiencing a bug where, occasionally and unpredictably,  when we send someone to the swell hosted checkout we are getting a 404 error:

![image (7)](https://user-images.githubusercontent.com/3166892/213796806-fe656bd0-dfb7-413c-930b-a3ea51129193.png)

The reason I'm pretty sure this is a bug is because if we refresh the page, the checkout loads correctly. So the link is indeed correct and not expired.

I'm wondering if this is a known bug that the team is working on? Obviously, breaking the checkout flow is an extremely costly bug in an ecommerce store and its affecting our live store.

**Comment by greph** [comment]
Hi @vettloffah , I'm not seeing any internal issues filed. Can you help provide some additional info:
* accepted payment methods
* using a custom domain?
* still seeing the issue?
